developer/architecture/layout.md

```diff
@@ -1,12 +1,93 @@
+
+`layout`
+* This property's name is a bit misleading. Conceptually it designates a shop wide identifier for a collection of _Layouts_ (subsequently referred to only as **_Global Layout_**), because it covers potentially many routes (as opposed to _Layout_, which only refers to a specific _View_)
+
+`collection`
+* Refers to a mongo collection which holds the _workflow state_, if any.
+
+`structure`
+* Contains information about which components/widgets should be rendered when this _workflow process_ is applied by the Reaction router.
+* `structure.template`
+  * The name of the [Blaze](http://docs.meteor.com/#/full/blaze_render) template that renders the main content for this _workflow process_.
+* `structure.notFound`
+  * Name of Blaze template that gets rendered if `structure.template` does not exist (Should not be necessary).
+* `structure.headerLayout`
+  * The name of the React component that renders the header for this _workflow process_.
+* `structure.footerLayout`
+  * The name of the React component that renders the footer for this _workflow process_.
+
+`enabled`
+  * Wether this _workflow process_ should be enabled or not.
+
+### Workflow step properties
+`workflow`
+* Name for the workflow. Correlates with a _View_ specified in registry. See also the [Routes section](https://docs.reactioncommerce.com/reaction-docs/master/plugin-routes-6)
+
+`container`
+* The DOM element where the rendered template should go into.
+
+`template`
+  * The name of the [Blaze](http://docs.meteor.com/#/full/blaze_render) template that renders the content for this _workflow step_.
+
+`enabled`
+  * Wether this _workflow step_ should be enabled or not
+
+### Notices
+* At any given time, only one _Global Layout_ can be active. See also `layout` property above.
+
+* At any given time - within the active _Global Layout_ - only one of the related workflow processes can be active.
+
+### Implementation details:
+
+* Layouts are meant to be created using the Package Registry. During initialization, the _workflow processes_ are cloned to the Shops collection which is where the application refers to them at runtime. Contrary, the _workflow steps_ are not cloned to the Shops collection.
+
+* It's perfectly legal to have a workflow process **without** a corresponding workflow step. This just means that the rendered _Container_ does not have children that are dependent on the state of the _workflow process_.
+
+* The _workflow process_ will be rendered with the `ReactionLayout` function in /imports/plugins/core/router/lib/router.js.
+
+* The _workflow steps_ will be rendered with the `reactionTemplate` Meteor helper.
+
+* The Registry serves as the "source of truth", allowing customizations to be made directly to the individual shops. In the case where we have two _workflow processes_ with the same key structure (`layout`, `workflow`), the array is loaded in reverse order, so that the newest layout will be used.
```

Comment:
I believe this has actually been superseded by the `priority` attribute.

Reply:
@zenweasel  Yep, I copied that verbatim from old docs. I would suggest to leave that part out completely.